datahub-web-react/src/app/entity/shared/containers/profile/header/EntityTabs.tsx

```diff
@@ -25,23 +25,22 @@ const Tab = styled(Tabs.TabPane)`
 `;
 
 export const EntityTabs = <T,>({ tabs, selectedTab }: Props) => {
-    const { entityData } = useEntityData();
+    const { entityData, loading } = useEntityData();
     const routeToTab = useRouteToTab();
     const baseEntity = useBaseEntity<T>();
 
+    const visibleTabs = tabs.filter((tab) => tab.display?.visible(entityData, baseEntity));
+    const enabledAndVisibleTabs = visibleTabs.filter((tab) => tab.display?.enabled(entityData, baseEntity));
```

Comment:
Here we can just remove any checks for visible tabs - only visible tabs make there way in. 

We can just say "enabledTabs". Then we will route to the first enabled tab down below.